uber/config.py

```diff
@@ -73,9 +73,19 @@ def make_enum(self, enum_name, section, prices=False):
                 lookup[val] = desc
 
         enum_name = enum_name.upper()
-        setattr(self, enum_name + '_OPTS', opts)
-        setattr(self, enum_name + '_VARS', varnames)
-        setattr(self, enum_name + ('' if enum_name.endswith('S') else 'S'), lookup)
+        # When loading plugin's configs, we want to make sure we don't override existing enums
+        if hasattr(self, enum_name + '_OPTS'):
```

Comment:
To get around the issue of stuff like "which order do I load these plugins", we could also have a separate .ini file for just sideboard that has as little as possible in it.  

in the context of the configDB where everything is in the DB and not in the .ini, this file would specify probably only:
1) config DB / regular DB credentials
2) what order to load the plugins in

Stuff would get read out of the config DB, merged however we want, then processed through config.py